apps/crn-server/src/utils/analytics/collaboration.ts

```diff
@@ -0,0 +1,145 @@
+import { FetchUserCoproductionQuery } from '@asap-hub/contentful';
+import { TeamRole, UserCollaborationDataObject } from '@asap-hub/model';
+import { cleanArray, parseUserDisplayName } from '@asap-hub/server-common';
+
+export type EntityWithId = {
+  sys: {
+    id: string;
+  };
+};
+
+export type Author = {
+  id: string;
+  teams: EntityWithId[];
+  labs: EntityWithId[];
+};
+
+export const checkDifferentTeams = (
+  referenceTeam: EntityWithId,
+  testTeams: EntityWithId[],
+): boolean =>
+  !testTeams.find((testTeam) => testTeam.sys.id === referenceTeam.sys.id);
```

Comment:
small change, but I think it's more readable
```suggestion
  testTeams.every((testTeam) => testTeam.sys.id !== referenceTeam.sys.id)
```